src/geometry/MeshDrawMgr.cxx

```diff
@@ -84,16 +74,67 @@ MeshDrawMgr::~MeshDrawMgr()
     return;
 }
 
+void MeshDrawMgr::initVBO()
+{
+    vboArrayIndex = vboVTN.vboAlloc(vertexCount);
+    vboVTN.vertexData(vboArrayIndex, vertexCount, vertices);
+    vboVTN.textureData(vboArrayIndex, vertexCount, texcoords);
+    vboVTN.normalData(vboArrayIndex, vertexCount, normals);
+
+    for (int lod = 0; lod < lodCount; lod++)
+    {
+        const DrawLod& drawLod = drawLods[lod];
+        for (int set = 0; set < drawLod.count; set++)
+        {
+            const DrawSet& drawSet = drawLod.sets[set];
+            const int cmdCount = drawSet.count;
+            GLuint  indices[8192];
+            GLuint *indexP = indices;
+            int     fillP  = 0;
+            for (int i = 0; i < cmdCount; i++)
+            {
+                const DrawCmd& cmd = drawSet.cmds[i];
+                fillP += cmd.count;
+                if (fillP > 8192)
+                {
+                    std::cout << "MeshDrawMgr cmdSet " << fillP << " is too high" << std::endl;
+                    abort();
+                }
+                if (cmd.indexType == GL_UNSIGNED_SHORT)
+                {
+                    GLushort *shortP = (GLushort *)cmd.indices;
+                    for (int j = 0; j < cmd.count; j++)
+                        *indexP++ = (GLuint)*shortP++;
```

Comment:
for fun, consider
```c++
    auto src = static_cast<GLushort*>(cmd.indices);
    indexP = std::transform(src, src+cmd.count, indexP, [](auto v) { return v; });
```
(although I think `copy` would work)

Reply:
copy works